packages/legend-query/src/components/QueryBuilderValueSpecificationEditor.tsx

```diff
@@ -53,20 +60,68 @@ import {
   PRIMITIVE_TYPE,
   TYPICAL_MULTIPLICITY_TYPE,
   VariableExpression,
+  SimpleFunctionExpression,
 } from '@finos/legend-graph';
 import { getMultiplicityDescription } from './shared/QueryBuilderUtils';
+import { FormControlLabel, Radio, RadioGroup } from '@mui/material';
```

Comment:
As mentioned before, let's expose these via `@finos/legend-art` in a make-sense way.